docs/usage/nfd-master.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "NFD-Master"
+layout: default
+sort: 3
+---
+
+# NFD-Master
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
```

Comment:
probably worth to remove since the list is empty

Reply:
K, fixed